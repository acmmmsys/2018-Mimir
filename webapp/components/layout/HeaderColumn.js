import React, { Component } from 'react';
import { Grid } from 'semantic-ui-react'

import 'style/layout/GridLayout.scss';

class HeaderColumn extends Component {
    render() {
        return (
            <Grid.Column className='grid-column'>
                {this.props.children}
            </Grid.Column>
        )
    }
}

export default HeaderColumn;