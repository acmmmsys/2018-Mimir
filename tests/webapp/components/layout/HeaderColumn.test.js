import React from 'react';
import HeaderColumn from '../../../../webapp/components/layout/HeaderColumn'
import renderer from 'react-test-renderer';

let wrapper;

test('HeaderColumn does render', () => {
    wrapper = shallow(
        <HeaderColumn>

        </HeaderColumn>
    );
    expect(wrapper).toMatchSnapshot();
});
