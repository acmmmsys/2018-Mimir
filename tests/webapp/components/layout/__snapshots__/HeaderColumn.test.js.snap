// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderColumn does render 1`] = `
<GridColumn
  className="grid-column"
/>
`;
